devicetypes/heltun/heltun-hls01-switch.src/heltun-hls01-switch.groovy

```diff
@@ -0,0 +1,304 @@
+/**
+ *  Copyright 2021 Sarkis Kabrailian
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+metadata {
+	definition (name: "HELTUN HLS01 Switch", namespace: "HELTUN", author: "Sarkis Kabrailian", cstHandler: true) {
+		capability "Energy Meter"
+		capability "Power Meter"
+		capability "Switch"
+		capability "Temperature Measurement"
+		capability "Voltage Measurement"
+		capability "Configuration"      
+		capability "Health Check" 
+		capability "Refresh"
+		//zw:Ls2a type:1000 mfr:0344 prod:0004 model:000A ver:1.05 zwv:7.11 lib:03 cc:5E,55,98,9F,6C,22 sec:85,59,8E,86,72,5A,73,87,32,70,25,31,81,71,7A
+        //cc:"0x5E, 0x55, 0x98, 0x9F, 0x6C, 0x22", sec:"0x85, 0x59, 0x8E, 0x86, 0x72, 0x5A, 0x73, 0x87, 0x32, 0x70, 0x25, 0x31, 0x81, 0x71, 0x7A" (NOT WORKING)
+		fingerprint mfr: "0344", prod: "0004", inClusters:"0x5E,0x85,0x59,0x8E,0x55,0x86,0x72,0x5A,0x73,0x81,0x87,0x98,0x9F,0x6C,0x70,0x25,0x31,0x32,0x71,0x22,0x7A", deviceJoinName: "HELTUN Switch" //model: "000A"
```

Comment:
can you not define the fingerprint using the mfr/prod/model version instead?

Reply:
If you use just those three fields I expect you will have more success

Reply:
@greens mfr, prod, model are the same as in another reviewed DTH-> #76161, so as You recommended before ->  inClusters are needed here to distinguish those devices

Reply:
@greens Both devices share the same mfr/prod/model -> #76161, so additional fields are needed here to distinguish those devices (we already talked about it internally some time ago )